test/import/FSHImporter.ValueSet.test.ts

```diff
@@ -0,0 +1,552 @@
+        const valueSet = result.valueSets.get('ZooVS');
+        expect(valueSet.components.length).toBe(1);
+        assertValueSetFilterComponent(valueSet.components[0], 'ZOO', undefined, []);
+        expect(loggerSpy.getLastMessage()).toMatch(/"=".*string/);
+        expect(loggerSpy.getLastMessage()).toMatch(/File: Zoo\.fsh.*Line: 3\D/s);
+      });
+
+      it('should parse a value set that uses filter operator is-a', () => {
+        const input = `
+        ValueSet: AllUrsinesVS
+        * codes from system ZOO where code is-a #bear "Bear"
+        `;
+        const result = importText(input, 'Ursines.fsh');
+        expect(result.valueSets.size).toBe(1);
+        const valueSet = result.valueSets.get('AllUrsinesVS');
+        expect(valueSet.components.length).toBe(1);
+        assertValueSetFilterComponent(valueSet.components[0], 'ZOO', undefined, [
+          {
+            property: VsProperty.CODE,
+            operator: VsOperator.IS_A,
+            value: new FshCode('bear', undefined, 'Bear')
+              .withLocation([3, 49, 3, 60])
+              .withFile('Ursines.fsh')
+          }
+        ]);
+      });
+
+      it('should log an error when the is-a filter has a non-code value', () => {
+        const input = `
+        ValueSet: AllUrsinesVS
+        * codes from system ZOO where code is-a "Bear"
+        `;
+        const result = importText(input, 'Ursines.fsh');
+        expect(result.valueSets.size).toBe(1);
+        const valueSet = result.valueSets.get('AllUrsinesVS');
+        expect(valueSet.components.length).toBe(1);
+        assertValueSetFilterComponent(valueSet.components[0], 'ZOO', undefined, []);
+        expect(loggerSpy.getLastMessage()).toMatch(/"is-a".*code/);
+        expect(loggerSpy.getLastMessage()).toMatch(/File: Ursines\.fsh.*Line: 3\D/s);
+      });
+
+      it('should parse a value set that uses filter operator descendent-of', () => {
+        const input = `
+        ValueSet: AllFelinesVS
+        * codes from valueset ZooVS where code descendent-of ZOO#cat
+        `;
+        const result = importText(input, 'Felines.fsh');
+        expect(result.valueSets.size).toBe(1);
+        const valueSet = result.valueSets.get('AllFelinesVS');
+        expect(valueSet.components.length).toBe(1);
+        assertValueSetFilterComponent(
+          valueSet.components[0],
+          undefined,
+          ['ZooVS'],
+          [
+            {
+              property: VsProperty.CODE,
+              operator: VsOperator.DESCENDENT_OF,
+              value: new FshCode('cat', 'ZOO', undefined)
+                .withLocation([3, 62, 3, 68])
+                .withFile('Felines.fsh')
+            }
+          ]
+        );
+      });
+
+      it('should parse a value set that uses filter operator descendent-of, but spelled correctly', () => {
+        const input = `
+        ValueSet: AllFelinesVS
+        * codes from valueset ZooVS where code descendant-of ZOO#cat
```

Comment:
I think I'm a bit confused about this test. Is this supposed to indicate that `descendent-of` and `descendant-of` are both valid? If that's the case, could you just rename the test to clarify that it's an alternate spelling? If that's not the case, could you explain what is going on here?

Reply:
Yes, both spellings are valid. I'll clarify that in the test name.

Reply:
Perfect, thanks!